uint32 no_seri_control_unit

string rfid_tid

uint8 golongan_kendaraan
uint8 jenis_kendaraan
string plat_no_rss
string plat_no_anpr

uint32 saldo
uint32 tarif

uint8 no_gardu_entrance
uint8 no_gerbang_entrance
uint8 no_gardu_exit
uint8 no_gerbang_exit
uint8 entrance_year
uint8 entrance_month
uint8 entrance_day
uint8 entrance_hour
uint8 entrance_minute
uint8 entrance_second

string hash
